addons/web/static/src/views/fields/selection/json_selection_field.js

```diff
@@ -0,0 +1,41 @@
+import { Component } from "@odoo/owl";
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { selectionField, SelectionField } from "@web/views/fields/selection/selection_field";
+
+export class JsonSelectionField extends Component {
```

Comment:
could you add a comment above which explains the expected format of the json values?